__mocks__/mock_node.js

```diff
@@ -53,7 +64,7 @@ switch (process.env.CONFIG_NAME) {
     // Dev live is internally connected, but all external connections are mocked
     //
     console.warn('--- Injecting external TC mocks ---');
-    useSomeMocks("googleapis", "google-auth-library");
+    useSomeMocks(["googleapis", "google-auth-library"]);
```

Comment:
Misread "useSomeSocks" and was a bit puzzled 😁

Reply:
For the warm days.  Cold days it's `useAllSocks`!